README.md

```diff
@@ -1,10 +1,14 @@
 # Azure Spring Apps Sample - Simple Todo App
 
+There are 2 branches about this repository:
+1. [quickstart](https://github.com/Azure-Samples/ASA-Samples-Web-Application/blob/quickstart/README.md). This branch is used to teach you to run [spring web application](https://spring.io/web-applications) in [Azure Spring Apps](https://learn.microsoft.com/en-us/azure/spring-apps/overview) (ASA) in a short time.
+2. [reference-architecture](https://github.com/Azure-Samples/ASA-Samples-Web-Application/blob/reference-architecture/README.md). This branch give the reference architecture (best practice) of running spring web application in Azure Spring Apps.
```

Comment:
FYI: 
I'll do these things after this PR merged:
1. Create the 2 branches mentioned here.
2. Set `quickstart` as the default branch.
3. Delete main branch.